warp/warp_client.go

```diff
@@ -16,6 +16,7 @@ var _ WarpClient = (*warpClient)(nil)
 
 type WarpClient interface {
 	GetSignature(ctx context.Context, messageID ids.ID) ([]byte, error)
+	GetAggregateSignature(ctx context.Context, messageID ids.ID, quorumNum uint64) ([]byte, error)
```

Comment:
Do we intend to expose this as an api?

Reply:
Yup this is available via `warp_service.go` and is intended for private use only.